english/plugh/plugh.md

```diff
@@ -1 +1 @@
-Suddenly, you find yourself on a hill looking at a small building.  What a relief!
\ No newline at end of file
+Suddenly, you find yourself on a hill looking at a small building.  What a relief! Click [Hope](https://github.com/udacity/create-your-own-adventure/blob/master/english/disney/disney.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.